src/http-client.ts

```diff
@@ -28,8 +28,8 @@ export const post = async (
 export const fetchUnfucked = async (
   uri: string,
   method = "GET",
-  headers: Record<string, string> = {},
-  body: Record<string, unknown> = null,
+  headers: Record<string, string> | null = {},
```

Comment:
I don't think this needs to be `null`. Default headers are an empty object.

Reply:
The `get` and `post` functions can send `null` as headers